pkg/controller/controller.go

```diff
@@ -275,15 +275,30 @@ func (s *Service) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest
 			if !ok {
 				return nil, status.Errorf(codes.InvalidArgument, "replication enabled but no volume group prefix specified in storage class")
 			}
+
 			rpo, ok := params[s.WithRP(KeyReplicationRPO)]
 			if !ok {
-				return nil, status.Errorf(codes.InvalidArgument, "replication enabled but no RPO specified in storage class")
+				// If Replication mode is ASYNC and there is no RPO specified, returning an error
+				if repMode == "ASYNC" {
```

Comment:
Hi @rajendraindukuri , Can we define "ASYNC" & "ZERO" as a constant? What do you think?